lib/clone.js

```diff
@@ -14,6 +14,10 @@ function clone(x) {
 		return x;
 	}
 
+	if (typeof x === 'object' && x !== null && (x instanceof String || x instanceof Number || x instanceof Boolean)) {
```

Comment:
I think we can simplify slightly: `instanceof` is sufficient, and the previous `if` rules out `null` and non-objects.

```suggestion
	if (x instanceof String || x instanceof Number || x instanceof Boolean) {
```